static/sharing.js

```diff
@@ -28,7 +28,6 @@ var _ = require('underscore');
 var options = require('./options');
 var Components = require('./components');
 var url = require('./url');
-var ga = require('./analytics');
```

Comment:
Do we lose analytics of this completely? Not a biggie, I don't know we've ever looked at them...

Reply:
It was called at opening the popup, which is just a bootstrap menu now. We could add it to the menu item click if you want.

Reply:
re-added this